STC/scss/_variables.scss

```diff
@@ -0,0 +1,47 @@
+$colors: ('white':#fff,'black':#000,'purple':#4F008C,'dark-purple':#420374,'coral':#FF375E,
+'grey':#F3F3F1,'grey-text': #8E9AA0,'dark-grey':#D8D8D833,'grey-row':#F6F6F6,
+);
+
+ @font-face {
```

Comment:
you can name font family as same and make it globaly and the browser will detect it